<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>镭射自动分区转换程序 V20251024</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/icon.ico</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>60</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnLoadPrg">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>载入源程序...</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="maximumSize">
         <size>
          <width>450</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <property name="title">
         <string>参数设置</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="1">
            <widget class="QDoubleSpinBox" name="optThreshold">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="decimals">
              <number>1</number>
             </property>
             <property name="minimum">
              <double>0.100000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
             <property name="value">
              <double>5.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>对位点防反阈值：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>对位点显示大小：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="optMarkerSize">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="suffix">
              <string/>
             </property>
             <property name="minimum">
              <number>-1</number>
             </property>
             <property name="maximum">
              <number>10</number>
             </property>
             <property name="singleStep">
              <number>1</number>
             </property>
             <property name="value">
              <number>8</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>加工孔显示大小：</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="optHoleSize">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
             <property name="value">
              <number>4</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>显示分区边界线：</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QRadioButton" name="isShowFenquRect">
               <property name="text">
                <string>是</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="isHideFenquRect">
               <property name="text">
                <string>否</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnShowPlot">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>45</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>刷新分区
示意图</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="maximumSize">
         <size>
          <width>450</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>当前选定孔信息</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="1">
            <widget class="QLineEdit" name="fenquNo">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_20">
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>刀具编号</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="posX">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_8">
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>坐标X</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_19">
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>分区编号</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_11">
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>分区对位点：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_7">
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>坐标Y</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLineEdit" name="posY">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLineEdit" name="toolNo">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="markerTable">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>280</width>
             <height>150</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>320</width>
             <height>150</height>
            </size>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustToContents</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <row>
            <property name="text">
             <string notr="true">1</string>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </row>
           <row>
            <property name="text">
             <string>2</string>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </row>
           <row>
            <property name="text">
             <string>3</string>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </row>
           <row>
            <property name="text">
             <string>4</string>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </row>
           <column>
            <property name="text">
             <string>X坐标</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Y坐标</string>
            </property>
           </column>
           <item row="0" column="0">
            <property name="text">
             <string/>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </item>
           <item row="0" column="1">
            <property name="text">
             <string/>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </item>
           <item row="1" column="0">
            <property name="text">
             <string/>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </item>
           <item row="1" column="1">
            <property name="text">
             <string/>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </item>
           <item row="2" column="0">
            <property name="text">
             <string/>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </item>
           <item row="2" column="1">
            <property name="text">
             <string/>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </item>
           <item row="3" column="0">
            <property name="text">
             <string/>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </item>
           <item row="3" column="1">
            <property name="text">
             <string/>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="graphLayout">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnSavePrg">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>输出分区程序...</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
